modules/bringauto_sysroot/default_test.go

```
@@ -0,0 +1,260 @@
+	if err != nil {
+		t.Errorf("can't delete sysroot dir - %s", err)
+	}
+}
+
+func clearSysroot() error {
+	sysrootPath := defaultSysroot.GetSysrootPath()
+	return os.RemoveAll(filepath.Dir(sysrootPath))
+}
+
+func setupPackageFiles() error {
+	os.Mkdir(pack1Name, 0755)
```

golangci-lint (1.62.2)

Error return value of `os.Mkdir` is not checked

(errcheck)